alphamountain/package.yaml

```diff
@@ -0,0 +1,99 @@
+id: alphamountain
+name: alphaMountain
+author: Tenzir
+author_icon: https://github.com/tenzir.png
+package_icon: |
+  https://raw.githubusercontent.com/tenzir/library/main/alphamountain/package.png
+
+description: |
+  [alphaMountain](https://alphamountain.ai) provides up-to-date domain and IP
+  intelligence for cybersecurity investigational and protection platforms.  
+
+  This package provides the `alphamountain-threats` context for enrichment and
+  pipelines that constantly update the feed.
+
+inputs:
+  license:
+    name: License key
+    description: |
+      Your alphaMountain batch API key.
+    type: string
+    default: 00000000-0000-0000-0000-000000000000
```

Comment:
Is this a sane default? Or no default at all?

Reply:
Ideally this would be a hint, but this is better than nothing.